applications/rag/README.md

```diff
@@ -41,6 +51,11 @@ gcloud container node-pools create g2-standard-24 --cluster <cluster-name> \
  --node-locations $REGION-a,$REGION-b --location=$REGION
 ```
 
+3. Ensure your k8s client is using the correct cluster by running:
+```
+gcloud container clusters get-credentials $CLUSTER_NAME --location $CLUSTER_REGION
+```
+
 4. Enable additional required APIs:
```

Comment:
Can you delete this? we do it by default now: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/modules/cloudsql/main.tf#L18

Reply:
Done.